up-l1/README.adoc

```diff
@@ -288,9 +290,9 @@ We can get the message types by reorganizing the table above to the following on
 |===
 | Message Type | Possible resource_id combinations {src_resource_id, sink_resource_id}
 
-| Publish      | {[8000-FFFE], None}
+| Publish      | {[8000-FFFE], None}, {FFFF, None}
```

Comment:
No, we do not listen for all published events. This cannot be supported

Reply:
Removed